### PR TITLE
feat(support-universal-runtime): adopt all changes needed for create-helix-service to create universal runtime functions by default

### PR DESCRIPTION
**Pull request by trieloff**
BREAKING CHANGE: switch from openwhisk-action-builder (targeting Adobe I/O Runtime) to helix-deploy (targeting Universal Runtime)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,11 @@
   "dependencies": {
     "@adobe/create-helix-library": "1.4.6",
     "@adobe/helix-epsagon": "1.6.0",
+    "@adobe/helix-fetch": "^2.1.7",
     "@adobe/helix-status": "9.1.0",
     "@adobe/openwhisk-action-logger": "2.4.1",
     "@adobe/openwhisk-action-utils": "4.4.1",
+    "dotenv": "^8.2.0",
```
Comment (tripodsan) [inline package.json:38]: pin

File: package.json [hunk 2]
```diff
@@ -31,9 +31,11 @@
   "dependencies": {
     "@adobe/create-helix-library": "1.4.6",
     "@adobe/helix-epsagon": "1.6.0",
+    "@adobe/helix-fetch": "^2.1.7",
```
Comment (tripodsan) [inline package.json:34]: pin

hint: when you use `npm add -E @adobe/helix-fetch`, it uses the exact version....

File: template/package.json [hunk 3]
```diff
@@ -12,10 +12,10 @@
...
-    "deploy": "wsk-builder -v --deploy --test=/_status_check/healthcheck.json",
-    "deploy-sequences": "wsk-builder --no-build -no-hints -l latest -l major -l minor",
-    "deploy-ci": "wsk-builder -v --deploy --test=/_status_check/healthcheck.json --pkgVersion=ci$CIRCLE_BUILD_NUM -l ci"
+    "build": "hedy -v",
+    "deploy": "hedy -v --deploy --test=/_status_check/healthcheck.json",
+    "deploy-sequences": "hedy --no-build -no-hints -l latest -l major -l minor",
+    "deploy-ci": "hedy -v --deploy --test=/_status_check/healthcheck.json --pkgVersion=ci$CIRCLE_BUILD_NUM -l ci"
   },
   "wsk": {
     "name": "helix-services/service@${version}"
```
Comment (tripodsan) [inline template/package.json:21]:
```suggestion
    "name": "helix-services/service@${version}",
    "memory": 256
```

File: template/package.json [hunk 4]
```diff
@@ -47,6 +47,7 @@
     "codecov": "3.8.1",
     "commitizen": "4.2.3",
     "cz-conventional-changelog": "3.3.0",
+    "dotenv": "^8.2.0",
```
Comment (tripodsan) [inline template/package.json:50]: pin

File: test/post-deploy-utils.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
```
Comment (tripodsan) [inline test/post-deploy-utils.js:2]:
```suggestion
 * Copyright 2021 Adobe. All rights reserved.
```

**Review by tripodsan — APPROVED** [review]
basically ok...

File: test/post-deploy-utils.js (hunk 5 above)
Comment (trieloff) [reply test/post-deploy-utils.js:2]: I think you created that file originally in 2020, it has only been copied and tweaked since then.

**Review by trieloff — COMMENTED** [review]
.

File: template/package.json (hunk 4 above)
Comment (trieloff) [reply template/package.json:51]:
```suggestion
    "dotenv": "8.2.0",
```

File: package.json (hunk 2 above)
Comment (trieloff) [reply package.json:34]:
```suggestion
    "@adobe/helix-fetch": "2.1.7",
```

File: package.json (hunk 1 above)
Comment (trieloff) [reply package.json:38]:
```suggestion
    "dotenv": "8.2.0",
```

**Pull request merged**